Prometheus/Exemplar.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Globalization;
+
+namespace Prometheus;
+
+public static class Exemplar
```

Comment:
RFC: Helper types to ensure the Exemplar atoms are captured efficiently for encoding.

TODO:

* Add empty/whitespace check for label and value (verify with spec).